### DISCUSSION TITLE
Thumbnail images not showing up in Assets

### DISCUSSION THREAD

**Discussion by larsjanssen6** [statamic/cms#6772] [Troubleshooting]
Hi, 

I'm using s3 for storing my images. That's working however they do not show up when I view the assets (thumbnails). 

The url seems to be wrong as well, it's: 

https://mydomain.nl/cp/thumbnails/c251ZmZlbGJveF9zdG9yYWdlOjoxMV8yNl8tX1AzXzEyMDB4MTIwMC0xNjU5MDQyODAwLndlYnA=/small

But I would expect a link to digital ocean (where my bucket is).

What could be wrong here? On my blog etc. they are displayed correctly.

**Comment by duncanmcclean** [comment]
I think those URLs are correct as Statamic will resize the images with Glide. Are you getting any errors from those image URLs?

**Comment by larsjanssen6** [comment]
@duncanmcclean thanks for your reply. I only see this:

When I visit a url it keeps loading: https://snuffelbox.nl/cp/thumbnails/c251ZmZlbGJveF9zdG9yYWdlOjoxMC5wbmc=/small

![Screenshot 2022-09-23 at 11 37 21](https://user-images.githubusercontent.com/7254997/191933243-ccf98987-4d71-428d-9784-e3bedfc9931e.png)

I'm not seeing any errors in my laravel log 🤔

**Reply by duncanmcclean** [reply]
If you open up your console, do you see any errors?

**Reply by larsjanssen6** [reply]
Not seeing any error there.

**Reply by jasonvarga** [reply]
Is it a 500 error? What's in the logs?

**Reply by larsjanssen6** [reply]
Hi Jason, I'm not seeing anything in the logs. This is an example image, it's just black https://snuffelbox.nl/cp/thumbnails/c251ZmZlbGJveF9zdG9yYWdlOjoxMjAweDEyMDAtaG9uZGVuYnVnZ3ktbWluLTE2NjIyMzU5MjUuanBn/small. No idea what is happening here.

**Reply by jasonvarga** [reply]
We can't see that url because you need to be logged into the cp.

However you're saying it's black, but in the screenshots earlier, it shows they are broken. So which is it?

If you have your devtools open to the network tab and you visit that image url, what response code do you see?d

**Reply by larsjanssen6** [reply]
I see this error: `Failed to load resource: net::ERR_HTTP2_PROTOCOL_ERROR` in my console when I visit that ^ link. The images show up broken in my media library in statamic.

**Reply by jasonvarga** [reply]
What kind of web server are you using?

**Reply by jasonvarga** [reply]
and does that still happen if you use a different browser?

**Reply by larsjanssen6** [reply]
I'm using Laravel forge with digital ocean. It's an nginx server. And the problem occurs in every browser.

**Reply by jasonvarga** [reply]
Have you done any customizations to the nginx config?

**Reply by larsjanssen6** [reply]
It looks like this:

```
# FORGE CONFIG (DO NOT REMOVE!)
include forge-conf/domain.nl/before/*;

server {
    listen 443 ssl http2;
    listen [::]:443 ssl http2;
    server_name .domain.nl domain.de;
    server_tokens off;
    root /home/forge/domain.nl/public;

    # FORGE SSL (DO NOT REMOVE!)

    add_header X-Frame-Options "SAMEORIGIN";
    add_header X-XSS-Protection "1; mode=block";
    add_header X-Content-Type-Options "nosniff";

    index index.html index.htm index.php;

    charset utf-8;

    # FORGE CONFIG (DO NOT REMOVE!)
    include forge-conf/domain.nl/server/*;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location = /favicon.ico { access_log off; log_not_found off; }
    location = /robots.txt  { access_log off; log_not_found off; }

    access_log off;
    error_log  /var/log/nginx/domain.nl-error.log error;

    error_page 404 /index.php;

    location ~ \.php$ {
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_pass unix:/var/run/php/php8.1-fpm.sock;
        fastcgi_index index.php;
        include fastcgi_params;
    }

    location ~ /\.(?!well-known).* {
        deny all;
    }
}

# FORGE CONFIG (DO NOT REMOVE!)
include forge-conf/domain.nl/after/*;

```
